src/router.ts

```diff
@@ -969,6 +969,7 @@ export async function handleDisclaimer(
   const slackLobbyMessageParams = {
     parentMessageTs: userInfo[userInfo.activeChannelId],
     channel: userInfo.activeChannelId,
+    isAutomatedMessage: true,
```

Comment:
This isn't for an automated message necessarily, right? Looks like it's used below to relay the voter's message

Reply:
fixed!